staging/src/k8s.io/kubectl/pkg/drain/drain.go

```diff
@@ -241,10 +250,11 @@ func (d *Helper) evictPods(pods []corev1.Pod, policyGroupVersion string, getPodF
 			if err != nil {
 				errors = append(errors, err)
 			}
-		case <-globalTimeoutCh:
-			return fmt.Errorf("drain did not complete within %v", globalTimeout)
+		default:
```

Comment:
Do we need this default statement?  Doesn't it just re-enter the loop and check again?

Reply:
I'm not 100% sure.  I want to say it was blocking on reading from the channel without another case, I'll try to mock up a test locally.